### PR TITLE
Support new variables like server root url or client IP address in email template (fixes #22)

### LINKED ISSUE #22: Add server url to list of template variables

### PR DESCRIPTION
**Pull request by leplatrem**
Fixes #22

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -141,6 +141,9 @@ The template string can have placeholders:
 * ``uri``
 * ``action``
 * ``timestamp``
+* ``root_url``
```
Comment (Natim) [inline README.rst]: Why not `server_url`?

Comment (leplatrem) [inline README.rst]: I was wondering if we should include `/v1/` prefix ...

Comment (Natim) [inline README.rst]: Yes I think we should 👍

Comment (leplatrem) [inline README.rst]: We do, that's why I called it `root_url` :)

**Pull request merged**